site/src/pages/WorkspaceBuildPage/WorkspaceBuildPageView.tsx

```diff
@@ -212,6 +219,7 @@ const ScrollArea: FC<HTMLProps<HTMLDivElement>> = () => {
 	// Issue: https://github.com/coder/coder/issues/9687
 	// Reference: https://stackoverflow.com/questions/43381836/height100-works-in-chrome-but-not-in-safari
 	const contentRef = useRef<HTMLDivElement>(null);
+	const [height, setHeight] = useState<CSSProperties["width"]>("100%");
```

Comment:
Tiny thing, but could we set the type to be based on `CSSProperties["height"]>`?

Reply:
lol yes, idk why I used width 🤦